src/Altinn.Platform/Altinn.Platform.Authentication/Altinn.Platform.Authentication.Tests/Mocks/EnterpriseUserAuthenticationServiceMock.cs

```diff
@@ -56,13 +58,27 @@ public async Task<HttpResponseMessage> AuthenticateEnterpriseUser(EnterpriseUser
                     Content = new StringContent(credentialsJson.ToString(), Encoding.UTF8, "application/json")
                 };
 
-                var result = await _client.SendAsync(request).ConfigureAwait(false);
                 result.StatusCode = HttpStatusCode.OK;
-                result.Content = GetEnterpriseUserContent();
+                result.Content = GetEnterpriseUserContent(credentials.UserName);
 
                 return result;
             }
-             else
+            else if (credentials.UserName == "ValidUser2" && credentials.Password == "Valid:Password")
+            {
+                string credentialsJson = JsonSerializer.Serialize(credentials);
+                var request = new HttpRequestMessage
+                {
+                    Method = HttpMethod.Post,
+                    RequestUri = new Uri(_settings.BridgeAuthnApiEndpoint + "enterpriseuser"),
+                    Content = new StringContent(credentialsJson.ToString(), Encoding.UTF8, "application/json")
+                };
+
+                result.StatusCode = HttpStatusCode.OK;
+                result.Content = GetEnterpriseUserContent(credentials.UserName);
```

Comment:
This could perhaps be a method to reduce code duplication.

Reply:
As most of this was not needed and the rest was just different for each path I just left it as is.